packages/Webkul/API/Http/Controllers/Shop/ProductController.php

```diff
@@ -34,7 +34,7 @@ public function __construct(ProductRepository $productRepository)
      */
     public function index()
     {
-        return ProductResource::collection($this->productRepository->getAll(request()->input('category_id')));
+        return ProductResource::collection($this->productRepository->getAll(request()->input('categories')));
```

Comment:
Endpoint once implemented can never be changed. Unless and until some major release has been done.

Reply:
Alright should I leave it at category_id, but let it be used as category_id=1,2,3 too?

Reply:
I reverted some of the changes and left the functionality only:
https://github.com/bagisto/bagisto/pull/4617/commits/1714ea6ee8e7f1e7f1e72dc244760cc29e8f1956